build.gradle

```diff
@@ -35,7 +35,7 @@ repositories {
 }
 
 dependencies {
-    compileOnly 'com.google.firebase:firebase-messaging:[10.2.1, )'
```

Comment:
revert back to this: compileOnly 'com.google.firebase:firebase-messaging:[10.2.1, )'

Reply:
I just fixed it